src/Select/style.css

```diff
@@ -0,0 +1,108 @@
+.ant-select {
+  @apply font-inter;
+  @apply bg-white rounded-lg flex;
+
+  &-single:not(.ant-select-customize-input) {
+    .ant-select-selector {
+      @apply px-3 rounded-lg;
+
+      .ant-select-selection-item {
+        @apply m-auto;
+      }
+    }
+  }
+
+  &-single {
+    .ant-select-show-arrow {
+      .ant-select-selection-item {
+        @apply m-auto;
+      }
+    }
+  }
+
+  &:not(.ant-select-customize-input) {
+    .ant-select-selector {
+      @apply w-full !important;
+      @apply rounded-lg;
+    }
+  }
+
+  &:not(.ant-select-customize-input):hover {
+    .ant-select-selector {
+      @apply border-neutral-300;
+    }
+  }
+
+  &-focused:not(.ant-select-disabled).ant-select:not(.ant-select-customize-input) {
+    .ant-select-selector {
+      @apply border-neutral-300 shadow-none;
+    }
+  }
+
+  &-arrow {
+    @apply mr-1;
+  }
+
+  &-multiple {
+    .ant-select-selector {
+      @apply px-2 rounded-lg;
+      padding-top: 5px;
```

Comment:
We must not have this kind of padding. If this padding exists in our design, we must inform the designer to change to tailwindcss padding.